python_modules/libraries/dagster-dbt/dagster_dbt_tests/cloud_v2/test_resources.py

```diff
@@ -209,9 +209,13 @@ def my_dbt_cloud_assets(context: AssetExecutionContext, dbt_cloud: DbtCloudWorks
     asset_materialization_events = result.get_asset_materialization_events()
     asset_check_evaluation = result.get_asset_check_evaluations()
 
-    # materializations are successful outputs, asset check evaluations are not outputs
+    # materializations and check results are successful outputs
     outputs = [event for event in result.all_events if event.is_successful_output]
-    assert len(outputs) == 8
+    assert len(outputs) == 28
+
+    # materialization outputs have metadata, asset check outputs don't
+    outputs_with_metadata = [output for output in outputs if output.step_output_data.metadata]
```

Comment:
Feels weird to just arbitrarily check metadata; we should be checking for the particular metadata we care about

Reply:
Done in [f026025](https://github.com/dagster-io/dagster/pull/28907/commits/f0260253a87398784b204dd0b5e51c0592c7b051)